renter/rpc.go

```diff
@@ -276,14 +276,14 @@ func (s *Session) FormContract(renterKey types.PrivateKey, renterFunds, hostColl
 		return rhp.Contract{}, types.Transaction{}, errors.New("host signature is invalid")
 	}
 
-	// add the host's additions to the transaction.
+	// add the host's additions to the transaction
```

Comment:
my (admittedly eclectic) commenting style is:

- treat "simple" comments (up to 1 sentence) as sentence fragments: no initial capital, no period
- multi-sentence comments should use proper capitalization and punctuation
- liberally abuse semicolons, em-dashes, and parentheticals in order to prevent a single sentence from becoming multiple